--- ClamAv.cc.orig 2019-02-09 22:44:27.770356182 -0500
+++ ClamAv.cc.patched 2019-02-09 23:41:41.963098901 -0500
@@ -44,7 +44,12 @@
     // We assume that cl_*() functions used here are threadsafe.

     const char *virname = 0;
- const int eScanResult = cl_scanfile(answer.fileName.c_str(), &virname, 0, engine, CL_SCAN_STDOPT);
+ static struct cl_scan_options options;
+
+ memset(&options, 0, sizeof(struct cl_scan_options));
+ options.parse |= ~0; /* enable all parsers */
+
+ const int eScanResult = cl_scanfile(answer.fileName.c_str(), &virname, 0, engine, &options);

     switch (eScanResult) {
     case CL_CLEAN: