--- ClamAv.cc	2015-11-08 18:07:35.000000000 +0000
+++ ClamAv.cc.new	2025-07-07 10:46:45.000000000 +0000
@@ -44,8 +44,13 @@
     // We assume that cl_*() functions used here are threadsafe.
 
     const char *virname = 0;
-    const int eScanResult = cl_scanfile(answer.fileName.c_str(), &virname, 0, engine, CL_SCAN_STDOPT);
 
+    static struct cl_scan_options options = {};
+    {
+        options.parse |= ~0; // enable all parsers
+    }
+    const int eScanResult = cl_scanfile(answer.fileName.c_str(), &virname, 0, engine, &options);
+    
     switch (eScanResult) {
     case CL_CLEAN:
         answer.statusCode = Answer::scClean;
@@ -58,7 +63,7 @@
 
     default:
         answer.statusCode = Answer::scError;
-        answer.errorMsg = cl_strerror(eScanResult);
+        answer.errorMsg = cl_strerror((cl_error_t)eScanResult);
     }
 }
 
@@ -71,7 +76,7 @@
     if (!initialized) {
         const int ret = cl_init(CL_INIT_DEFAULT);
         if (ret != CL_SUCCESS)
-            Throw("Can't initialize libclamav: ", cl_strerror(ret));
+            Throw("Can't initialize libclamav: ", cl_strerror((cl_error_t)ret));
         initialized = true;
     }
 
@@ -136,13 +141,13 @@
         /* load all available databases from default directory */
         int ret = cl_load(cl_retdbdir(), engine, &sigs, CL_DB_STDOPT);
         if (ret != CL_SUCCESS)
-            Throw("cl_load: ", cl_strerror(ret));
+            Throw("cl_load: ", cl_strerror((cl_error_t)ret));
 
         // printf("Loaded %u signatures from %s\n", sigs, cl_retdbdir());
 
         // build engine
         if ((ret = cl_engine_compile(engine)) != CL_SUCCESS)
-            Throw("Database initialization error: ", cl_strerror(ret));;
+            Throw("Database initialization error: ", cl_strerror((cl_error_t)ret));;
 
         memset(&dbstat, 0, sizeof(struct cl_stat));
         cl_statinidir(cl_retdbdir(), &dbstat);
