--- squidclamav.conf	2017-01-17 09:47:04.398126061 +0100
+++ squidclamav.conf.new	2017-01-17 10:29:56.338094368 +0100
@@ -15,7 +15,7 @@
 
 # When a virus is found then redirect the user to this URL. If this directive
 # is disabled squidclamav will use c-icap error templates to report issues.
-redirect http://proxy.domain.dom/cgi-bin/clwarn.cgi
+# redirect http://proxy.domain.dom/cgi-bin/clwarn.cgi
 
 # Path to the squiGuard binary if you want URL filtering, note that you'd better
 # use the squid configuration directive 'url_rewrite_program' instead.
@@ -40,7 +40,7 @@
 # preserve backward compatibility but you must desactivate this feature if you
 # don't use trustclient with hostname in the regexp or if you don't have a DNS
 # on your network. Disabling it will also speed up squidclamav.
-dnslookup 1
+dnslookup 0
 
 # Enable / Disable Clamav Safe Browsing feature. You mus have enabled the
 # corresponding behavior in clamd by enabling SafeBrowsing into freshclam.conf
